pkg/services/alerting/notifier_test.go

```diff
@@ -18,18 +19,21 @@ import (
 )
 
 func TestNotificationService(t *testing.T) {
-	testRule := &Rule{
-		ID:            1,
-		DashboardID:   1,
-		PanelID:       1,
-		OrgID:         1,
-		Name:          "Test",
-		Message:       "Something is bad",
-		State:         models.AlertStateAlerting,
-		Notifications: []string{"1"},
-	}
+	testRule := newTestRule("Test", "Something is bad")
 	evalCtx := NewEvalContext(context.Background(), testRule)
 
+	testRuleTemplated := newTestRule("Test latency {{.quantile}}", "Something is bad on instance {{ .instance }}")
```

Comment:
Total nit, but it's less confusing to the reader if the template style is consistent (otherwise it can look as if whitespace is significant):
```suggestion
	testRuleTemplated := newTestRule("Test latency {{.quantile}}", "Something is bad on instance {{.instance}}")
```